app/Console/Command/AddLabelToItems.php

```diff
@@ -0,0 +1,170 @@
+            $count = $this->eventsSearchResultsGenerator->count($query);
+            if ($this->askConfirmation($input, $output, $newLabel, $count, $itemType)) {
+                $results = $this->eventsSearchResultsGenerator->search($query);
+                $this->addLabelToItems(
+                    $output,
+                    $count,
+                    $results,
+                    $itemType,
+                    $newLabel
+                );
+            }
+        }
+
+        if ($itemType->sameAs(ItemType::place())) {
+            $count = $this->placesSearchResultsGenerator->count($query);
+            if ($this->askConfirmation($input, $output, $newLabel, $count, $itemType)) {
+                $results = $this->placesSearchResultsGenerator->search($query);
+                $this->addLabelToItems(
+                    $output,
+                    $count,
+                    $results,
+                    $itemType,
+                    $newLabel
+                );
+            }
+        }
+
+        if ($itemType->sameAs(ItemType::organizer())) {
+            $count = $this->organizersSearchResultsGenerator->count($query);
+            if ($this->askConfirmation($input, $output, $newLabel, $count, $itemType)) {
+                $results = $this->organizersSearchResultsGenerator->search($query);
+                $this->addLabelToItems(
+                    $output,
+                    $count,
+                    $results,
+                    $itemType,
+                    $newLabel
+                );
+            }
+        }
+
+        return self::SUCCESS;
+    }
+
+    private function askConfirmation(InputInterface $input, OutputInterface $output, Label $label, int $count, ItemType $itemType): bool
+    {
+        return $this
+            ->getHelper('question')
+            ->ask(
+                $input,
+                $output,
+                new ConfirmationQuestion(
+                    sprintf('This action will add the label %s to %u %ss, continue? [y/N] ', $label->getName()->toString(), $count, $itemType->toString()),
+                    true
+                )
+            );
+    }
+
+    private function addLabelToItems(OutputInterface $output, int $count, Generator $results, ItemType $itemType, Label $label): void
+    {
+        $progressBar = new ProgressBar($output, $count);
+        $exceptions = [];
+        foreach ($results as $id => $result) {
+            try {
+                if ($itemType->sameAs(ItemType::organizer())) {
+                    $labelCommand = new OrganizerAddLabel($id, $label);
+                } else {
+                    $labelCommand = new OfferAddLabel($id, $label);
+                }
+                $this->commandBus->dispatch(
+                    $labelCommand
+                );
+            } catch (Exception $exception) {
+                $exceptions[$id] = 'Item with id: ' . $id . ' caused an exception: ' . $exception->getMessage();
```

Comment:
I wonder if it's better to directly show the error, if we run the command in attended mode you can force quit it after the first failure. Otherwise, it could fail for all items but run for a very long time before showing any errors.